### ISSUE TITLE
SVG Tag not supported

### ISSUE THREAD

**Issue by pauloafernandes-82** [spipu/html2pdf#534]
It appears the renderer does not support '<svg>' or `<use>` tags. I'm hoping there's future support for this as it's becoming much more common. 
![Screen Shot 2020-01-17 at 13 23 44](https://user-images.githubusercontent.com/20706067/72640044-a81c6980-392c-11ea-9f4c-c588c05aff01.png)

**Comment by spipu** [comment]
you are right, it is not implemented. But if you want, you can implement it and make a MR :)
